main.tf

```diff
@@ -73,6 +73,10 @@ resource "aws_ecs_service" "service" {
   desired_count                     = var.ecs_service_desired_count
   health_check_grace_period_seconds = var.health_check_grace_period_seconds
 
+  ordered_placement_strategy {
```

Comment:
Using this strategy, tasks will be spread evenly across instances 
https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-placement-strategies.html